cfme/services/myservice/ssui.py

```diff
@@ -229,6 +229,36 @@ def retire(self):
     assert view.notification.assert_message("{} was retired.".format(self.name))
 
 
+@MiqImplementationContext.external_for(MyService.service_start, ViaSSUI)
+def service_start(self):
```

Comment:
Optional: This can be done as one method with start/stop parameter

Reply:
I would say it should be a private method and public methods should call it with a respectful parameter.